js/models/conversations.js

```diff
@@ -38,6 +38,20 @@
     deleteAttachmentData,
   } = window.Signal.Migrations;
 
+  // Possible conversation friend states
+  const FriendStatusEnum = Object.freeze({
+    // New conversation, no messages sent or received
+    none: 0,
+    // Have received a friend request, waiting to accept/decline
+    pendingIn: 1,
+    // Have send a friend request, waiting for response
+    pendingOut: 2,
```

Comment:
thoughts on renaming to the following:
`pendingIn` -> `pendingAction`
`pendingOut` -> `pendingResponse`

Reply:
Yeah that makes sense to me, I'll push another commit